proto/satellite.proto

```diff
@@ -264,3 +264,47 @@ message SatOpRow {
     // check nulls_bitmask to differentiate between the two
     repeated bytes values = 2;
 }
+
+// A migration message, originating in Postgres, captured via event triggers,
+// propated to electric, converted from postgres to the equivalent sqlite
+// statement and inserted into the replication stream
+//
+// Each migration message includes the sql strings to execute on the satellite
+// client as well as metadata information about the resulting structure of the
+// changed tables.
+message SatMigration {
```

Comment:
Looks like that this message is going to be send as an individual message in the stream, so it needs to be added here:
https://github.com/electric-sql/typescript-client/blob/main/src/util/proto.ts
(lines 25/36)

Reply:
gotcha, thanks